force-app/main/universalLearner/lwc/learnerCourseVisualizer/learnerCourseVisualizer.html

```diff
@@ -0,0 +1,33 @@
+<template>
+    <template if:false={shouldDisplayCourses}>
+        <!-- Display spinner -->
+        <lightning-spinner alternative-text={labelReference.spinnerLoadingAltText} variant="brand" size="medium">
+        </lightning-spinner>
+    </template>
+    {jsonDebug}
+    <lightning-card
+        icon-name="custom:custom83"
+        title={learnerCourseViewModel.schemaLabels.assertion.achievement.achievementType}
+        if:true={shouldDisplayCourses}
+    >
+        <ul class="slds-accordion">
+            <template for:each={learnerCourseViewModel.courses} for:item="learnerCourse">
```

Comment:
Do we need a message if there's no courses not display?

Reply:
Right now, no. We may choose not to display the entire component at all, but that's TBD